cmd/agent/openstack_iid_attestor/main.go

```diff
@@ -61,25 +55,6 @@ func (p *IIDAttestorPlugin) Configure(ctx context.Context, req *spi.ConfigureReq
 		return nil, fmt.Errorf("failed to decode configuration file: %v", err)
 	}
 
-	if req.GlobalConfig == nil {
-		return nil, errors.New("global configuration is required")
-	}
-	if req.GlobalConfig.TrustDomain == "" {
-		return nil, errors.New("trust_domain is required")
-	}
-
-	p.mtx.Lock()
-	defer p.mtx.Unlock()
-
-	meta, err := p.getMetadataHandler()
-	if err != nil {
-		return nil, fmt.Errorf("failed to retrieve openstack metadta: %v", err)
-	}
-
-	p.metaData = meta
-	config.trustDomain = req.GlobalConfig.TrustDomain
-	p.config = config
-
 	return &spi.ConfigureResponse{}, nil
 }
```

Comment:
The agent plugin does not use any configurations, including the global configurations.
So it would be nice to update this func to the following definition like the [Agent NodeAttestor "join_token"](https://github.com/spiffe/spire/blob/v0.10.1/pkg/agent/plugin/nodeattestor/jointoken/join_token.go#L34-L35) .

```go
func (p *IIDAttestorPlugin) Configure(ctx context.Context, req *spi.ConfigureRequest) (*spi.ConfigureResponse, error) {
	return &spi.ConfigureResponse{}, nil
}
```

Reply:
fix: 0fe4d7e